src/icicle.js

```diff
@@ -191,7 +192,10 @@ export default Kapsule({
 
     // Entering
     const newCell = cell.enter().append('g')
-      .attr('class', 'node')
+      .attr('class', d => [
```

Comment:
This should be added to the `allCells` section below instead. So it reacts to changes in case the `nodeClassName` changes after a cell is created.

Reply:
updated